keyboards/monsgeek/m2/info.json

```
@@ -0,0 +1,263 @@
+{
+    "manufacturer": "MonsGeek",
+    "keyboard_name": "M2",
+    "maintainer": "jonylee@hfd",
+    "bootloader": "wb32-dfu",
+    "diode_direction": "ROW2COL",
+    "dynamic_keymap": {
+        "layer_count": 6
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "indicators": {
+        "caps_lock": "A15",
+        "num_lock": "C10"
+    },
```

migrated from config.h
```suggestion
    },
    "qmk": {
        "locking": {
            "enabled": true,
            "resync": true
        }
    },
```